CC/Sounder/BaseRadioSet.cc

```diff
@@ -173,37 +178,46 @@ BaseRadioSet::BaseRadioSet(Config* cfg)
         tddConfStr += "]}";
         std::cout << tddConfStr << std::endl;
 #endif
-        for (size_t i = 0; i < bsRadios[0].size(); i++) {
-            SoapySDR::Device* dev = bsRadios[0][i]->dev;
-            dev->writeSetting("TX_SW_DELAY", "30"); // experimentally good value for dev front-end
-            dev->writeSetting("TDD_MODE", "true");
-            dev->writeSetting("TDD_CONFIG", tddConfStr);
-        }
+        // write TDD schedule and beacons to FPFA buffers only for Iris
+        if (!kUseUHD) {
+            for (size_t i = 0; i < bsRadios[0].size(); i++) {
+                SoapySDR::Device* dev = bsRadios[0][i]->dev;
+                dev->writeSetting("TX_SW_DELAY", "30"); // experimentally good value for dev front-end
+                dev->writeSetting("TDD_MODE", "true");
+                dev->writeSetting("TDD_CONFIG", tddConfStr);
+            }
```

Comment:
I am thinking your BS implementation can also work for Faros. In that case, we can condition this on `hw_framer` which is currently only used for clients, instead of `kUseUHD`.

Reply:
Although I understand that should be separately tested for Faros,